tests/Console/E2ETest.php

```diff
@@ -40,7 +40,7 @@ protected function setUp()
         // Without overcommit this test fails with `proc_open(): fork failed - Cannot allocate memory`
         if (strpos(PHP_OS, 'Linux') === 0 &&
             is_readable('/proc/sys/vm/overcommit_memory') &&
-            file_get_contents('/proc/sys/vm/overcommit_memory') === 2) {
+            intval(file_get_contents('/proc/sys/vm/overcommit_memory')) === 2) {
```

Comment:
`(int) X` is more performant that `intval(X)` (reported by `EA Extended Ultimate`)

![intval](https://user-images.githubusercontent.com/3725595/42367644-906ca30a-810d-11e8-886c-1d59bb380f8b.png)

Reply:
That part is very far from being a hot code, but whatever.

Reply:
We have the `modernize_types_casting ` php-cs fixer rule ( part of `@Symfony:risky`) which should auto fix this.

Reply:
@sanmai we are already using it as part of `@Symfony:risky`